src/main.js

```diff
@@ -122,7 +123,7 @@ export default class Auth {
 			let data = await response.json();
 
 			// If the response returned an error, try to get a Firebase error code/message.
-			// Sometimes the error codes are joined with an explanation, we don't need that(its a bug).
+			// Sometimes the error codes are joined with an explanation, we don't need that.
```

Comment:
The comment that’s it’s a bug is important so that future contributors know why the code does what it does. If it gets fixed in the future it will affect the way we want to handle it